integrations/vscode/src/start-server.ts

```diff
@@ -46,7 +45,7 @@ async function startServer(outputChannel: OutputChannel): Promise<boolean> {
     }
   }
   if (nodeVersion.stdout) {
-    outputChannel.appendLine(stripAnsi(nodeVersion.stdout));
+    outputChannel.appendLine(ignoreBellPrefix(nodeVersion.stdout));
```

Comment:
This appears to be an unintentional revert of https://github.com/fwouts/previewjs/commit/cb84e188db1a8378511bac64fd3cede6767ac547#diff-aa8c66b56caa464dff97471b0d6d1ac0556adbd8427ef22cae8083c33c9db8c7.

These changes and below need to be reverted, only the port (lines 8 + 15) needs to change in this PR.